homeassistant/components/nordpool/__init__.py

```diff
@@ -5,13 +5,25 @@
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.typing import ConfigType
 from homeassistant.util import dt as dt_util
 
 from .const import DOMAIN, PLATFORMS
 from .coordinator import NordPoolDataUpdateCoordinator
+from .services import async_setup_services
 
 type NordPoolConfigEntry = ConfigEntry[NordPoolDataUpdateCoordinator]
 
+CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the Habitica service."""
```

Comment:
```suggestion
    """Set up the Nordpool service."""
```

Reply:
LOL. That's how it goes when you copy (steal) from others 😁